Sources/XcodeGraph/Models/Target.swift

```diff
@@ -97,7 +97,8 @@ public struct Target: Equatable, Hashable, Comparable, Codable, Sendable {
         onDemandResourcesTags: OnDemandResourcesTags? = nil,
         metadata: TargetMetadata = .metadata(tags: []),
         type: TargetType = .local,
-        packages: [AbsolutePath] = []
+        packages: [AbsolutePath] = [],
+        buildableFolders: [BuildableFolder] = []
```

Comment:
can you move this property below `sources` ?, it will be better when both are next to each other. Then as a user I don't have to jump around to see which files or folder are part of target

Reply:
note that `XcodeGraph` is not the abstraction used in `Project.swift`. We have an extra layer that sits in-between.

I don't have a strong preference here, but agree that in `ProjectDescription`, it should be close to `sources` and `resources`.